app/src/main/java/com/example/habitsmasher/ui/profile/ProfileFragment.java

```diff
@@ -68,68 +51,122 @@ public View onCreateView(@NonNull LayoutInflater inflater,
-                Bundle bundle = new Bundle();
-                bundle.putString("FollowType", "Following");
-                // navigate to following fragment
-                navigateToFragmentWithAction(R.id.action_navigation_notifications_to_followListFragment, bundle);
-            }
-        });
+    @Override
+    public void onDestroyView() {
+        super.onDestroyView();
     }
 
     /**
-     * This is the click listener for the followers button
-     * @param numberOfFollowers
+     * Not needed
      */
-    private void setClickListenerForFollowersButton(Button numberOfFollowers) {
-        numberOfFollowers.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                // this is a bundle to tell the fragment whether its following or followers
-                Bundle bundle = new Bundle();
-                bundle.putString("FollowType", "Followers");
-                // navigate to followers fragment
-                navigateToFragmentWithAction(R.id.action_navigation_notifications_to_followListFragment, bundle);
-            }
-        });
+    @Override
+    protected Query getListFromFirebase() {
+
+        return null;
     }
 
     /**
-     * This function navigates to another fragment with a given bundle
-     * @param actionId The navigation action
-     * @param bundle The bundle being passed in
+     * Not needed
      */
-    private void navigateToFragmentWithAction(int actionId, Bundle bundle) {
-        NavController controller = NavHostFragment.findNavController(_fragment);
-        controller.navigate(actionId, bundle);
+    @Override
+    protected void populateList(Query query) {
     }
 
+    /**
+     * Not needed
+     */
     @Override
-    public void onDestroyView() {
-        super.onDestroyView();
+    public void onStart() {
+        super.onStart();
+        //_habitEventItemAdaptor.startListening();
+    }
+
+    /**
+     * Not needed
+     */
+    @Override
+    public void onStop()
+    {
+        super.onStop();
+        //_habitEventItemAdaptor.stopListening();
+    }
+
+    /**
+     * Not needed
+     */
+    @Override
+    protected void openAddDialogBox() {
+        //not needed
+    }
+
+    /**
+     * Not needed
+     */
+    @Override
+    public void openEditDialogBox(int position) {
+        //not needed
+    }
+
+    /**
+     * Not needed
+     */
+    @Override
+    protected void openViewWindowForItem(int position) {
     }
+
+    /**
+     * Not needed
+     */
+    @Override
+    protected void initializeRecyclerView(LinearLayoutManager layoutManager, View view) {
+
+    }
+
+    /**
+     * Not needed
+     */
+    @Override
+    public void updateListAfterAdd(User addedObject) {
+
+    }
+
+    /**
+     * Not needed
+     */
+    @Override
+    public void updateListAfterEdit(User editedObject, int pos) {
+
+    }
+
+    /**
+     * Not needed
```

Comment:
I wonder if there's a better way to do this than simply overriding and then saying not needed?